.gitignore

```diff
@@ -26,3 +26,4 @@ community/server-plugin-test/neo4j-home
 enterprise/server-enterprise/neo4j-home
 integrationtests/data
 dependency-reduced-pom.xml
+venv
```

Comment:
out of curiosity, can `venv` work with JDK installations?
I'm currently using https://www.jenv.be/, but if there is a better alternative, I'm eager to try :)

Reply:
venv I think is generated by IntelliJ for python. Haven't seen jenv before, I'm just swithcing between java versions using aliases "alias j8='export JAVA_HOME=`/usr/libexec/java_home -v 1.8`'"